src/server/status/index.js

```diff
@@ -1,6 +1,8 @@
 import _ from 'lodash';
 import ServerStatus from './server_status';
 import { join } from 'path';
+import Config from '../config/config';
```

Comment:
Ha, good catch.  Looks like I got lucky below with the argument being passed in.